CHANGELOG.md

```diff
@@ -33,6 +33,9 @@
 
   The `file::exists` and `file::readable` functions will no longer error when the file path is specified relative to a module and the file doesn't exist.
 
+* **`bolt inventory show -- detail` now displays all target aliases** ([#1379](https://github.com/puppetlabs/bolt/issues/1379))
```

Comment:
Can you rebase and put this in a new `Bolt Next` section?

Reply:
Updated!